docs/development_tips.md

```diff
@@ -22,6 +22,10 @@ If you made a change to a core integration and you want to test it in your produ
 
 The Home Assistant frontend caches aggressively, and as such, the first time you run Home Assistant with your new changes, you may not see the integration show up in the integration list. Check the logs to make sure there were no errors, and if not, perform a hard refresh of your browser window and try again; in many cases, that will resolve your issue.
 
+### Running pytests from within VSCode
+
+If you installed the required Python dependencies, you can run PyTests from within VSCode after some configuration. Via the VSCode side-menu you select "Testing" (icon: Labratory Flask) and select that the tests are in the folder "tests". After a while a treeview should appear with tests which can be executed via the "Run Test" button (icon: Play) next to the test names. 
```

Comment:
Please use tasks or testing as documented. The test feature of VSCode  you are trying to document will really not work well with the amount of tests we have and is well known to have issues for our codebase size.

../Frenck